packages/slice-machine/package.json

```diff
@@ -38,7 +38,6 @@
     "atob": "^2.1.2",
     "babel-plugin-module-resolver": "^4.0.0",
     "babel-plugin-prismjs": "^2.0.1",
-    "base64-img": "^1.0.4",
```

Comment:
this package is not used anymore ?

Reply:
couldn't find it any where when running.
`grep -irn "base64-img" server src start helpers hooks lib templates components pages`